docs/src/userguide.md

```diff
@@ -290,6 +301,20 @@ get_coeff(exy, [3,5])
 rationalize(ans)
 ```
 
+Similar to `Taylor1`, vector notation can be used to request specific
+coefficients of `HomogeneousPolynomial` or `TaylorN` objects. For `TaylorN`
+objects, the index refers to the degree of the `HomogeneousPolynomial`.
+In the case of `HomogeneousPolynomial` the index refers to the position
+of the hash table. The function [`show_monomials`](@ref) can be used to
+obtain the coefficient a specific monomial, given the degree of the
+`HomogeneousPolynomial`.
+
+```@repl userguide
+exy[8] # get the 8th order term
+show_monomials(8)
+exy[8][6] # get the 1st coeff of the 2th order term
```

Comment:
What about `exy[8, 6]`?

Reply:
I guess this could be possible, though I don't know how, since `TaylorN` are vectors. This currently throws a `BoundsError`; leaving it aside for the moment.

Incidentlally, your suggestion made me notice that the comment on that line is wrong; it should read "# get the 6th coeff of the 8th order term". I have corrected it.